### ISSUE TITLE
Dart + Electron

### ISSUE THREAD

**Issue by Xyene** [DMOJ/hephaestus#1]

**Comment by Xyene** [comment]
@quantum5

**Comment by Xyene** [comment]
There's a pub package for it: https://pub.dartlang.org/packages/dart_amqp

Admittedly since Dartium won't run in Electron, compilation would incur the wrath of `dart2js` speed.

**Comment by quantum5** [comment]
Let's just not worry about RabbitMQ and the stuff since it's really not that hard to parse the current protocol.

**Comment by quantum5** [comment]
RabbitMQ on a non-server seems really painful: not only do you have to install RabbitMQ, you will have to install Erlang. I would suggest alternative packet managers. It's a separate class already for the pluggability anyway.
